src/planscape/attributes/attribute_retrieval_testcase.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+from attributes.models import Attribute, AttributeRaster
+from conditions.raster_condition_retrieval_testcase import RasterRetrievalTestCase
+from django.contrib.gis.gdal import GDALRaster
+from planscape import settings
+
+
+class RasterAttributeRetrievalTestCase(RasterRetrievalTestCase):
```

Comment:
is it possible to define this in the _test.py file to make debugging easier? for the file in conditions, i can see wanting to keep it separate because we need to import it in other apps

Reply:
yup! actually: I worked on a follow-up PR and realized a distinct RasterAttributeRetrievalTestCase isn't even necessary.
Fixed.